src/SwedishBankAccounts.UnitTests/BankAccountNumberTests.cs

```diff
@@ -1,10 +1,8 @@
-﻿using System;
-
-namespace SwedishBankAccounts.UnitTests;
+﻿namespace SwedishBankAccounts.UnitTests;
 
 public class BankAccountNumberTests : UnitTests
 {
-    [Theory] 
+	[Theory] 
```

Comment:
Include InitOptions as an argument. Set all current InlineData to InitOptions.Strict. Add a new InlineData for a value that validates using Lax.